setup.cfg

```diff
@@ -34,7 +34,8 @@ install_requires =
     matplotlib
     torchvision
     scikit-learn
-    torchsde @ https://github.com/google-research/torchsde.git
+dependency_links=
+    https://github.com/google-research/torchsde.git
```

Comment:
I think u need to add the git+  prefix here. 
As per https://www.freecodecamp.org/news/how-to-use-github-as-a-pypi-server-1c3b0d07db2/

Reply:
Ooh I think I missed this. Thanks!